### PR TITLE
Send bug reports to GitHub

### PR DESCRIPTION
**Pull request by jkeenan**
Mail to 'perlbug@perl.org' now simply triggers a response redirecting
sender there.

### CODE REVIEW

File: dist/IO/lib/IO/Socket.pm [hunk 1]
```diff
@@ -916,8 +916,8 @@ L<IO::Socket::IP>
 
 =head1 AUTHOR
 
-Graham Barr.  atmark() by Lincoln Stein.  Currently maintained by the
-Perl Porters.  Please report all bugs to <perlbug@perl.org>.
+Graham Barr. atmark() by Lincoln Stein.  Currently maintained by the Perl
```
Comment (rjbs) [inline dist/IO/lib/IO/Socket.pm]:
```suggestion
Graham Barr.  atmark() by Lincoln Stein.  Currently maintained by the Perl
```

Extreme nitpick:  you've eliminate one double-space-after-full-stop in this paragraph, but not the others.  Let's just leave that be?

**Review by rjbs — APPROVED** [review]
I have provided only one very very small comment that needn't stand in the way — but would be easy to apply.  Looks good.

File: dist/IO/lib/IO/Socket.pm (hunk 1 above)
Comment (jkeenan) [reply dist/IO/lib/IO/Socket.pm]: Accepted.

**Comment by jkeenan** [comment]
Updated with one other 2-character correction.

**Review by toddr — APPROVED** [review]

**Comment by toddr** [comment]
I already point the cpan module at p5p so this is perfect. Thanks @jkeenan

**Pull request merged**